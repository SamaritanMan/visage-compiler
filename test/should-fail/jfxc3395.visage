/**
 * Regress test for JFXC-3395: An internal error has occurred in the Visage compiler
 *
 * @test/compile-error
 */

var s:String;
jfxc3395{s:""};
