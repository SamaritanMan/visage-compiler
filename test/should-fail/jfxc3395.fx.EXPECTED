test/should-fail/jfxc3395.fx:8: script variable s cannot be initialized inside object literal expression
jfxc3395{s:""};
         ^
1 error
